### DISCUSSION TITLE
data.db 文件大小12G

### DISCUSSION THREAD

**Discussion by nanqic** [AlistGo/alist#7316] [Q&A]
不知道是不是bug，运行了半年多，主要挂载wasabi，还有一百多行地址树，不知道什么原因啊，大家有遇到过这样的情况吗？

` -rw-r--r-- 1 root root  12G Oct 11 17:51 data.db `

配置都是用的默认的，运行在Ubuntu 22上。

**Comment by nanqic** [comment]
web打开挂在的目录，提示 failed select meta: database is locked

**Comment by Simonzhang8** [comment]
开索引了？

**Reply by nanqic** [reply]
是的，索引导致的吗？


搜索索引

数据库（非全文搜索）
自动更新索引
忽略路径
每行一条路径
最大索引深度
20
索引的最大深度
当前索引
对象计数:10113

上次完成时间:未知

**Reply by Simonzhang8** [reply]
索引太深文件太多 就会把data.db弄得非常大
之前我索引开太大了直接把服务器弄炸了都

**Comment by nanqic** [comment]
谢谢，可能是索引导致的，清除索引也没用，无奈只好用以前备份的data/覆盖了
